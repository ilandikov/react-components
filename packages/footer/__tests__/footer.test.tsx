/* External dependencies */
import { useStaticQuery } from 'gatsby';
import React from 'react';
import renderer from 'react-test-renderer';

/* Local dependencies */
import Footer from '../';

describe('Footer', () => {
  beforeEach(() => {
    useStaticQuery.mockImplementationOnce(() => ({
      site: {
        siteMetadata: {
          companyName: 'Company'
        }
      }
    }));
  });

  it('renders correctly', () => {
    const component = renderer.create(<Footer />);

    expect(component.toJSON()).toMatchSnapshot();
  });
});
