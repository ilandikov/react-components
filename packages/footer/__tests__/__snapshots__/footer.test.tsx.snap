// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer renders correctly 1`] = `
<p
  className=""
>
  © Company. Все права защищены.
</p>
`;
